### ISSUE TITLE
Issues with my dot.net installation 

### ISSUE THREAD

**Issue by arroyoshipp** [dotnet/core#3551]
Problem encountered on https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/install
Operating System: windows

Provide details about the problem you are experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

**Comment by choXberg** [comment]
I think you forgot to tell us more about the issues with the dotnet installation ;)

**Comment by arroyoshipp** [comment]
Iam.going to lay this out only once, here at Github since its where my issue too me.   A repository was created a few years back with my online credentials that repository still exists.  He has received all the credit for things that should have been mine.   I ended up on the streets in prison and being deported.  I am in MX, not in the US.  None of my devices have worked right since.   Any of you area welcome to look into the inside of my them.   Though the digital signature Is there the bulk of the benefits are not.   My social media my github, my azure my previous Google that I loved do much went bad as well.   I have gone through so many laptops and they have all ended up shell's without an OS or even the ability to watch a movIe.  My private network transmits a secure message to this person.   He knows everything I do.  I stopped using systems altogether.  The know community gave me a second life here in MX.  I figured I could start a new, I was wrong.  With privacy policies changing to accommodate abuse.  Things have gotten worse.   This is the final phase in completely taking away that which is most important. An Azure MS exchange certificate wss iSsued to me, but I didn't request it was and downloaded along with my Wim 7 OS
